cc-ansible

```diff
@@ -359,6 +359,8 @@ kolla_args+=(--globals="$CONFIG_DIR/globals.yml")
 kolla_args+=(--passwords="$CONFIG_DIR/passwords.yml")
 kolla_args+=(--extra node_custom_config="$CONFIG_DIR/node_custom_config")
 kolla_args+=(--extra cc_ansible_site_dir="$CONFIG_DIR")
+kolla_args+=(--extra deployment_dir="$DIR")
+kolla_args+=(--extra site_config_dir="$CC_ANSIBLE_SITE")
```

Comment:
how are these two values used? What about the case where the user has set `--site </path/to/site_config>` rather than setting the environment variable `CC_ANSIBLE_SITE`?

Reply:
I think you can do either way https://github.com/ChameleonCloud/chi-in-a-box/blob/master/cc-ansible#L192-L202.

both `deployment_dir` and `site_config_dir` are used for setting up a periodic job of mariadb_backup: https://github.com/ChameleonCloud/chi-in-a-box/blob/master/roles/chameleon_mariadb/tasks/deployment.yml

Reply:
thanks, looks good to me!